package.json

```diff
@@ -22,6 +22,7 @@
     "mongodb": "^3.5.9",
     "node-fetch": "^2.6.0",
     "pubnub": "^4.28.2",
+    "react-icons": "^3.10.0",
```

Comment:
As this is related only to client, install it `./client` folder. 
Remove it from the root package.